JS_2_names.js

```diff
@@ -0,0 +1,21 @@
+var arrUserNames = [];
+
+for (var i = 0; i < 5 ; i++) {
+  arrUserNames[i] = prompt('Введите имя пользователя');
+}
+
+var myName = prompt('Введите Ваше имя');
+
+var matchNames = 0;
+
+for (var i = 0; i < arrUserNames.length; i++) {
+  if (myName === arrUserNames[i]) {
+    matchNames += 1;
```

Comment:
You can replace counter with Boolean flag and break the loop if the match found.

Reply:
done